Evaluator/Strategies/market_making_startegy_evaluator.py

```diff
@@ -0,0 +1,53 @@
+"""
+OctoBot Tentacle
+
+$tentacle_description: {
+    "package_name": "OctoBot-Tentacles",
```

Comment:
Need to remove this line: package-name is auto generated by tentacle manager when installing (it's used to know the origin of a tentacle)